public/controllers/agent/agents-preview.js

```diff
@@ -92,6 +94,14 @@ export class AgentsPreviewController {
         !parameters.items.length > 0 && !parameters.filters.length;
     });
 
+    this.$scope.offsetTimestamp = (text, time) => {
```

Comment:
Move this method to an instance method.

```js
offSetTimestamp(text,time) {
 ...
}
```

Then, in the view, you can access it using `ctrl.offsetTimestamp`

Reply:
I removed the function from this controller because it wasn't being used.